core/Pos/Crypto/Signing/Types/Signing.hs

```diff
@@ -111,6 +111,12 @@ parseFullPublicKey s = do
     b <- Base64.decode s
     PublicKey <$> first fromString (CC.xpub b)
 
+-- | Parse 'PublicKey' from base16 encoded string.
+parseFullPublicKeyHex :: Text -> Either Text PublicKey
```

Comment:
No-no-no, please, let's stop it. I find it very bad that we don't have a single standard way of displaying a public key.
Base64 is currently used in JSON format. Base16 is currently used in scripts.
Unfortunately it seems that we can't change any of these two. And I propose to treat base64 as the standard way of displaying public keys and treat base16 as «only for scripts».
Nobody uses scripts nowadays and probably they won't be used for a long time, while canonical json contains genesis data which is quite useful in some cases.

Reply:
base16 is used in logs, which is why I thought it was the default. I'll change the language to use base64, but then it would be more convenient to change the logs as well.

Reply:
Ah, in `Buildable`, yes, I see. It's really bardaq :(